test/fixtures.coffee

```diff
@@ -874,22 +945,22 @@ sampleError = """
 """
 
 sampleHeaders = {
-"Content-Type": "application/json",
+"content-type": "application/json",
```

Comment:
I'll change it back to it's original value. And I'll make a new test with the lower-cased + upper-cased versions then. Would that be ok?